packages/ui/src/ui/store/selectors/global/index.ts

```diff
@@ -214,6 +214,11 @@ export const getGlobalAsideHeaderWidth = (state: RootState) => state.global.asid
 
 export const getGlobalYTAuthCluster = (state: RootState) => state.global.ytAuthCluster;
 
-export const getSettingsCluster = createSelector([getCluster], (cluster) => {
-    return userSettingsCluster ?? cluster;
-});
+export const getParamsCluster = (state: RootState): string => state?.global?.paramsCluster;
+
+export const getSettingsCluster = createSelector(
+    [getParamsCluster, getCluster],
+    (paramsCluster, cluster) => {
+        return userSettingsCluster || cluster || paramsCluster;
```

Comment:
Do we really need the change to fix the issue?

Reply:
`updateTitle` do nothing with settings, it cannot affect the issue.
There are many places where `getCluster` selector is use URL is not the best source of thruth.

Reply:
let's move `updateTitle`-call up